Topics/Development_Process/Build_Requirements/Introduction_to_make_and_Makefiles.md

```diff
@@ -0,0 +1,228 @@
+# Introduction to ```make``` and Makefiles
+
+## Table of contents
+
+### [Introduction](#introduction-1)
+
+### [Basic Makefile Characteristics](#basic-makefile-characteristics-1)
+
+### [Making a Makefile](#making-a-makefile-1)
+
+### [Advantages and Disadvantages to ```make``` and Makefiles](#advantages-and-disadvantages-to-make-and-makefiles-1)
+
+### [Additional Resources](#additional-resources-1)
+
+## Introduction
+
+#### What are ```make``` and Makefiles
+```make``` is a software build automation tool that builds programs and libraries, and determines what needs to be recompiled. It accomplishes this by using Makefiles that specify compilation targets and link their dependencies, and checking their last modified dates during compilation.
```

Comment:
What is the overall purpose of this guide? You should briefly explain it here.

Reply:
Good point, I'll add it in, thanks!